sample_apps/run_example.sh

```diff
@@ -228,8 +239,12 @@ function list_steps() {
             list_steps_for_app "${Steps_OE[@]}"
             ;;
 
+        "simple_app_under_gramine")
+            list_steps_for_app "${Steps[@]}"
```

Comment:
For now, this new-app shares the `Steps[]` array with those of `simple_app`.

I did not find any new steps needed for this new-app, so this sharing is ok for now.